assets/charts/control-plane/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -34,24 +11,19 @@ metadata:
     k8s-app: kube-apiserver
 spec:
   {{- if eq (int .Values.apiserver.replicas) 1 }}
-  replicas: 1
+  # Always run two replicas when it is single controller node.
+  replicas: 2
```

Comment:
This change shouldn't be here, commit should be re-organized to avoid adding this noise.